server.js

```diff
@@ -711,6 +711,19 @@ const envVarsCheck = callback => {
   }
 };
 
+const couchDbNoAdminPartyModeCheck = callback => {
+  const noAuthUrl = process.env.COUCH_URL.replace(/\/\/[^@]+@/, '//'),
+        protocol = noAuthUrl.split('://')[0];
+  const net = require(protocol);
+  net.get(noAuthUrl, err => {
+    if (err && err.statusCode === 401) {
+      callback();
+    } else {
+      callback(err || new Error('CouchDB is configured in admin-party mode (ie no auth required)'));
```

Comment:
Maybe create a medic-docs document for how to get out of admin-party mode and link from here? Or link to 3rd party docs if you can find something good enough.